File: Python-Basics/fishing_boat.py
budget = int(input())
season = input()
fishermen = int(input())

price_per_season = 0
discount = 0
total_price = 0

if season == "Spring":
    price_per_season = 3000

elif season == "Autumn":
    price_per_season = 4200

elif season == "Summer":
    price_per_season = 4200

elif season == "Winter":
    price_per_season = 2600

if fishermen <= 6:
    price_per_season *= 0.9

elif fishermen <= 11:
    price_per_season *= 0.85

elif fishermen > 12:
    price_per_season *= 0.75

if season != "Autumn" and fishermen %2 == 0:
    price_per_season *= 0.95

diff = abs(budget - price_per_season)

if budget >= price_per_season:
    print(f"Yes! You have {diff:.2f} leva left.")
else:
    print(f"Not enough money! You need {diff:.2f} leva.")

File: JS-Basics/Conditional Statements Advanced Exercise/fishing-boat.js
function fishing(input){
    let budget = Number(input[0]);
    let season = input[1];
    let fishers = Number(input[2]);
    let pricePerSeason = 0;
    let discount = 0;
    let totalPrice = 0;
    let diff = 0;

    if(season === "Spring"){
        pricePerSeason = 3000;
    }
    else if(season === "Summer" || season === "Autumn"){
        pricePerSeason = 4200;
    }
    else if(season === "Winter"){
        pricePerSeason = 2600;
    }

    if(fishers % 2 === 0 && season !== "Autumn"){
        discount += 0.05
    }
    if(fishers <= 6){
        discount += 0.1
    }
    else if(fishers >= 7 && fishers <= 11){
        discount += 0.15
    }
    else{
        discount += 0.25
    }

    totalPrice = pricePerSeason - (pricePerSeason * discount);
    diff = Math.abs(totalPrice - budget);

    if(budget >= totalPrice){
        console.log(`Yes! You have ${diff.toFixed(2)} leva left.`);
    }
    else{
        console.log(`Not enough money! You need ${diff.toFixed(2)} leva.`);
    }

}
fishing(["2000",
"Winter",
"13"])
;
